### PR TITLE
add contributors

### PR DESCRIPTION
**Pull request by tbo47**

### CODE REVIEW

**Review by sidharthv96 — APPROVED** [review]

**Review by aloisklink — APPROVED** [review]
You can also add our names to https://github.com/tbo47/dagre-es/blob/bae97cf3ffa06d721b1f3f6561826bbe39b14313/LICENSE.md?plain=1#L5

Although it would mean every single person that copies code from `dagre-es` would need to have a massive copyright header!

**Comment by tbo47** [comment]
> You can also add our names to
> 
> https://github.com/tbo47/dagre-es/blob/bae97cf3ffa06d721b1f3f6561826bbe39b14313/LICENSE.md?plain=1#L5
> 
> Although it would mean every single person that copies code from `dagre-es` would need to have a massive copyright header!

What about just leaving it like this:
```
Copyright (c) 2022-2024 dagre-es contributors
```

**Comment by aloisklink** [comment]
> What about just leaving it like this:

IANAL (I am not a lawyer), but I think under the MIT license, we might need to ask everybody that contributed to this repo to re-license it without the `Thibaut Lassalle` in the copyright statement. To be honest, I don't think anybody would mind (other than @tbo47 :laughing:), but it's not worth adding legal ammo, just in case somebody who contributed worked for @oracle or another litigious company.

Adding to it is fine, removing stuff from it is a bit more legally dubious.

**Comment by tbo47** [comment]
> > What about just leaving it like this:
> 
> IANAL (I am not a lawyer), but I think under the MIT license, we might need to ask everybody that contributed to this repo to re-license it without the `Thibaut Lassalle` in the copyright statement. To be honest, I don't think anybody would mind (other than @tbo47 😆), but it's not worth adding legal ammo, just in case somebody who contributed worked for @oracle or another litigious company.
> 
> Adding to it is fine, removing stuff from it is a bit more legally dubious.

Do you mind adding/removing what you think is right to this PR?

File: LICENSE.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Original dagre-d3 copyright: Copyright (c) 2013 Chris Pettitt
 Original dagre copyright: Copyright (c) 2012-2014 Chris Pettitt
 Original graphlib copyright: Copyright (c) 2012-2014 Chris Pettitt
 
-Copyright (c) 2022-2023 Thibaut Lassalle and dagre-es contributors
+Copyright (c) 2022-2024 dagre-es contributors
```
Comment (aloisklink) [inline LICENSE.md:5]: We could do:

```suggestion
Copyright (c) 2022-2024 Thibaut Lassalle, David Newell, Alois Klink, Sidharth Vinod and dagre-es contributors
```

Or, we could keep things simple, and just change the copyright year:

```suggestion
Copyright (c) 2022-2024 Thibaut Lassalle and dagre-es contributors
```

I believe both are fine from a legal standpoint.

**Comment by tbo47** [comment]
@aloisklink do you need more time for testing before I release a `7.0.11`?

**Comment by aloisklink** [comment]
> @aloisklink do you need more time for testing before I release a 7.0.11?

Yep, sorry.

I'm testing this with [mermaid-js/mermaid](https://github.com/mermaid-js/mermaid) and between dagre v7.0.10 and v7.0.11, I'm seeing a lot of visual differences:

Git-Graph-diagram-Should-render-subgraphs-with-title-margins-and-edge-labels diff

I haven't yet investigated exactly why there's differences, but I bet we've probably missed something in #34 that isn't causing an error, but is changing some behaviour somewhere.

**Comment by tbo47** [comment]
Maybe we have to do a lot more of defensive checks.

When I look at this code https://github.com/lodash/lodash/blob/main/src/has.ts
It is testing if the object is not null.

**Comment by tbo47** [comment]
I think I'm going to revert all the lodash related changes. I don't see the point of removing them and take the risk of having regressions. @aloisklink

**Pull request merged**